test/suites/clustering.sh

```diff
@@ -6,19 +6,21 @@ test_clustering_enable() {
   chmod +x "${LXD_INIT_DIR}"
   spawn_lxd "${LXD_INIT_DIR}" false
 
+  # Test specified core.https_address with no cluster.https_address
   (
     set -e
     # shellcheck disable=SC2034,SC2030
     LXD_DIR=${LXD_INIT_DIR}
 
+    lxc config show | grep "core.https_address" | grep -qE "127.0.0.1:[0-9]{4,5}$"
     # Launch a container.
     ensure_import_testimage
     lxc storage create default dir
     lxc profile device add default root disk path="/" pool="default"
     lxc launch testimage c1
 
     # Enable clustering.
-    lxc cluster enable node1
+    lxc cluster enable node1 | grep -q "Clustering enabled"
```

Comment:
What's the purpose of that?

Is there a case where `lxc cluster enable` will return 0 (success) and not have enabled clustering?

Reply:
I was just trying to be safe there. I can simplify it.

Reply:
Okay. I don't think this change adds anything and it makes the test a bit more brittle by making it dependent on that string as well as potentially making it succeed should that string be returned despite a failure.